### PR TITLE
Add example showing various escaping techniques

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: draft-ietf-cbor-update-8610-grammar.md [hunk 1]
```diff
@@ -299,9 +300,61 @@ It does require some care when copy-pasting into CDDL models from ABNF
...
+repertoire of a CDDL file itself to ASCII {{-ascii}}.
 
+~~~ cddl
+start = [a, b, c, x, y, z]
+
+; "🁁", DOMINO TILE HORIZONTAL-02-02, and
+; "⌘", PLACE OF INTEREST SIGN, in a text string:
+a = "D\u{6f}mino's \u{1F041} + \u{2318}"  ; \u{}-escape 3 chars
+b = "Domino's \uD83C\uDC41 + \u2318"      ; escape JSON-like
+c = "Domino's 🁁 + ⌘"                      ; unescaped
```
Comment (chrysn) [inline draft-ietf-cbor-update-8610-grammar.md]: At least GitHub's code viewer renders 🁁 wider than a single character width, offsetting the comments. If that happens with the IETF rendered view as well, 🂁 might avoid the issue.

File: draft-ietf-cbor-update-8610-grammar.md [hunk 2]
```diff
@@ -299,9 +300,61 @@ It does require some care when copy-pasting into CDDL models from ABNF
...
+
+; in a byte string given as text, the ' needs to be escaped:
+x = 'D\u{6f}mino\'s \u{1F041} + \u{2318}' ; \u{}-escape 3 chars
```
Comment (chrysn) [inline draft-ietf-cbor-update-8610-grammar.md]:
```suggestion
x = 'D\u{6f}mino\u{27}s \u{1F041} + \u{2318}' ; \u{}-escape 4 chars
```

might be a nice addition.

**Review by chrysn — APPROVED** [review]

File: draft-ietf-cbor-update-8610-grammar.md (hunk 1 above)
Comment (cabo) [reply draft-ietf-cbor-update-8610-grammar.md]: 🁁 is the result of a long journey to get a non-BMP character into RFC 9535.
🁳 probably would be accepted by the RFC editor as well, so I'll make this small esthetic change.

File: draft-ietf-cbor-update-8610-grammar.md (hunk 2 above)
Comment (cabo) [reply draft-ietf-cbor-update-8610-grammar.md]: I wondered about doing that.  This muddies the example a little bit.  But I did it anyway, for additional variety.

**Pull request merged**